# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ruby/ossl_x509store.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN ruby-1.8.2.orig/ext/openssl/ossl_x509store.c ruby-1.8.2/ext/openssl/ossl_x509store.c
--- ruby-1.8.2.orig/ext/openssl/ossl_x509store.c	2004-12-19 09:28:32.000000000 +0100
+++ ruby-1.8.2/ext/openssl/ossl_x509store.c	2005-07-14 21:22:03.000000000 +0200
@@ -538,7 +538,7 @@
 
     if(NIL_P(time)) {
 	GetX509StCtx(self, store);
-	store->flags &= ~X509_V_FLAG_USE_CHECK_TIME;
+	store->param->flags &= ~X509_V_FLAG_USE_CHECK_TIME;
     }
     else {
 	long t = NUM2LONG(rb_Integer(time));
